bin/summarize-user-events

```diff
@@ -30,12 +31,15 @@ class SummarizeUserEvents
 
   def matchers
     @matchers ||= [
-      EventSummarizer::ExampleMatcher.new
+      EventSummarizer::ExampleMatcher.new,
+      EventSummarizer::IdvMatcher.new
     ]
   end
 
   def run
     find_cloudwatch_events do |event|
+      Time.zone ||= 'America/New_York'
+      
```

Comment:
probably easier to set the timezone once, instead of for each event?
```suggestion
    Time.zone ||= 'America/New_York'

    find_cloudwatch_events do |event|      
```

Reply:
Done in #11627

Reply:
I ran into an issue with this--I think that when the Cloudwatch client uses multiple threads each _thread_ needs its time zone initialized.

Reply:
ah, I'll update it to use the configured zone.

Reply:
hmmm maybe we need to update the CW client to call that block back on the main thread only?